pkg/querier/batch/chunk_test.go

```diff
@@ -31,7 +31,7 @@ func mkChunk(t require.TestingT, from model.Time, points int) chunk.Chunk {
 	metric := model.Metric{
 		model.MetricNameLabel: "foo",
 	}
-	pc, err := promchunk.NewForEncoding(promchunk.DoubleDelta)
+	pc, err := promchunk.NewForEncoding(promchunk.Bigchunk)
```

Comment:
Shouldn't we test for each chunk encoding?

Reply:
We do in pkg/querier/querier_test.go; but will do it here too.